mtd-utils/build.sh

```diff
@@ -7,11 +7,12 @@ if [ "$(uname)" = "Darwin" ]; then
 
 	PREFIX_MTD_UTILS="${PREFIX_PROJECT}/mtd-utils"
 	PREFIX_MTD_UTILS_BUILD="${PREFIX_PROJECT}/_build/host-generic-pc/mtd-utils"
```

Comment:
maybe use `PREFIX_BUILD_HOST` (here and everywhere below also)?

Reply:
Good idea :+1:

Reply:
`PREFIX_BUILD_HOST` is being exported from `build.sh`, don't define it by yourself. 

Also - missed the line above.

Reply:
Thanks, done.